include/cpp-statsd-client/StatsdClient.hpp

```diff
@@ -39,29 +44,34 @@ class StatsdClient {
     //!@{
 
     //! Sets a configuration { host, port, prefix, batchsize }
-    void setConfig(const std::string& host, const uint16_t port, const std::string& prefix, const uint64_t batchsize = 0) noexcept;
+    void setConfig(const std::string& host,
+                   const uint16_t port,
+                   const std::string& prefix,
+                   const uint64_t batchsize = 0) noexcept;
```

Comment:
i dont believe i've changed this line, yet it seems the linter decided to. in other projects ive learned that i had to pin the version of clang-format because different versions of that applied the style differently and so people would be committing constant lint changes back and fourth. i can make a separate PR to pin clang-format and run lint as a part of the github actions